main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup
+
+url = 'https://pogoda.meta.ua/ua/Kyivska/Kyivskiy/Kyiv/'
+
+
+def temperature():
+    sourse = requests.get(url)
+    page = sourse.text
+    feaures = "html.parser"
+    soup = BeautifulSoup(page, feaures)
```

Comment:
`feaures` variable declaration is excessive. The below is more readable to me:

```suggestion
    soup = BeautifulSoup(page, "html.parser")
```